files/ddraw.ini

```diff
@@ -10,6 +10,33 @@
 ;XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 [Misc]
 
+; if IFACE_BAR_MODE=0 - the bottom of the map view window sits at the top of the IFACE Bar.
+; if IFACE_BAR_MODE=1 - the bottom of the map view window extends to the base of the screen and is overlapped by the IFACE Bar.
+IFACE_BAR_MODE=0
+
+;if IFACE_BAR_SIDE_ART=0 - Black, No Iface-bar side art used.
+;if IFACE_BAR_SIDE_ART=1 - Metal (grey) look Iface-bar side art used.
+;if IFACE_BAR_SIDE_ART=2 - Leather look Iface-bar side art used.
+;if IFACE_BAR_SIDE_ART=3-6 - Alternative Metal look Iface-bar side art used.
+;if IFACE_BAR_SIDE_ART=7 - Alternative Leather look Iface-bar side art used.
+;if IFACE_BAR_SIDE_ART=8 - Metal (brown) look Iface-bar side art used.
+IFACE_BAR_SIDE_ART=5
+
+
+;if IFACE_BAR_SIDES_ORI=0 - Iface-bar side graphics extend from the Iface-Bar to the Screen edges.
+;if IFACE_BAR_SIDES_ORI=1 - Iface-bar side graphics extend from the Screen edges to the Iface-Bar.
+IFACE_BAR_SIDES_ORI=1
+
+;This will increase the width of the interface bar expanding the area used to display text.
+;if IFACE_BAR_WIDTH=640 - Interface bar will remain at it's original width.
+IFACE_BAR_WIDTH=800
+
+;Adds Fallout 1 style travel/trail dots
+WorldMapTravelMarkers=0
+
+;Prevents pausing music between map areas with the same music
+GaplessMusic=0
+

```

Comment:
Maybe just keep it in the f2_res.ini until we have a new permanent home for these settings? Moving temporarily is just confusing.